extensions/positron-python/python_files/positron/positron_ipykernel/ui.py

```diff
@@ -216,12 +208,48 @@ def open(self, url, new=0, autoraise=True):
                     height=0,
                 ).dict(),
             )
-            return True
 
         for addr in _localhosts:
             if addr in url:
-                event = ShowUrlParams(url=url)
-                self._comm.send_event(name=UiFrontendEvent.ShowUrl, payload=event.dict())
+                is_plot = self._is_module_function("plotly.basedatatypes", "show")
+                if is_plot:
+                    if os.name == "nt":
+                        url = urlparse(url).netloc or urlparse(url).path
```

Comment:
Maybe this can be moved to `_send_show_html_event`?

Reply:
I've moved it into the new function